index.html

```diff
@@ -173,126 +173,364 @@ <h2>
+                  Queue an element task</a> on the 
+                  <a href="https://html.spec.whatwg.org/multipage/webappapis.html#user-interaction-task-source">
+                  User Interaction task source</a>, given this, to perform the following steps:
+                  <ol>
+                    <li>
+                      <a href=https://dom.spec.whatwg.org/#concept-event-fire">Fire an event</a> named
+                      <a>pointerlockerror</a> at <a>this</a>'s 
+                      <a href="https://dom.spec.whatwg.org/#concept-node-document">node document</a>.
+                    </li>
+                    <li>
+                      Return <a>a promise rejected with</a> a "{{WrongDocumentError}}" {{DOMException}}.
+                    </li>
+                  </ol>
+                </ol>
+              </li>
+              <li>
+                If the user agent's <a>pointer-lock target</a> is an element whose <a>shadow-including root</a> is
+                not equal to <a>this</a>'s <a>shadow-including root</a>, then:
+                <ol>
+                  1. <a href="https://html.spec.whatwg.org/multipage/webappapis.html#queue-an-element-task">
+                  Queue an element task</a> on the 
+                  <a href="https://html.spec.whatwg.org/multipage/webappapis.html#user-interaction-task-source">
+                  User Interaction task source</a>, given this, to perform the following steps:
+                  <ol type='i'>
+                    <li>
+                      <a href=https://dom.spec.whatwg.org/#concept-event-fire">Fire an event</a> named
+                      <a>pointerlockerror</a> at <a>this</a>'s 
+                      <a href="https://dom.spec.whatwg.org/#concept-node-document">node document</a>.
+                    </li>
+                    <li>
+                      Return <a>a promise rejected with</a> a "{{InvalidStateError}}" {{DOMException}}.
+                    </li>
+                  </ol>
+                </ol>
+              </li>
+              <li>
+                If the user agent's <a>pointer-lock target</a>'s <a>shadow-including
+                root</a> is equal to <a>this</a>'s <a>shadow-including root</a> follow these steps:
+                <ol>
+                  <li>
+                    If this request's <a>pointerLockOptions</a> are equivalent to the user agent's current
+                    <a>pointerLockOptions</a> then the <a>pointer-lock target</a>
+                    must be updated to this element, a <a>pointerlockchange</a>
+                    event sent, and the promise must <a>resolve</a>.
+                  </li>
+                  <li>
+                    If this request's <a>pointerLockOptions</a> are not equivalent to the user agent's current
+                    <a>pointerLockOptions</a> and this request's <a>pointerLockOptions</a> are not supported:
```

Comment:
"current pointerLockOptions" should refer to the user agent field discussed in "I think it would be better to define a specific field for the options..."